pkg/proxy/members_test.go

```diff
@@ -166,7 +166,7 @@ func (s *TestClusterServiceSuite) TestGetClusterAccess() {
 						fakeClient.MockGet = nil
 					}()
 					nsClient := namespaced.NewClient(fakeClient, commontest.HostOperatorNs)
-					svc := service.NewMemberClusterService(nsClient, sc)
+					svc := proxy.NewMemberClusters(nsClient, sc, commoncluster.GetMemberClusters)
```

Comment:
very minor - I guess we could rename also these `svc` which stands for service if I'm not mistaken

Reply:
good catch, I didn't pay much attention to the names in the test file.
renamed in 17a3ed7
Thanks